src/javascript/app/common/content_visibility.js

```diff
@@ -99,9 +103,7 @@ const ContentVisibility = (() => {
 
     const shouldShowElement = (
         attr_str,
-        current_landing_company_shortcode,
-        client_has_mt_company,
-        mt5fin_company_shortcode
+        params
```

Comment:
now that there are only two we can put it in a single line, e.g. `const shouldShowElement = (attr_str, params) => {`